modules/coreos-layering-configuring-on-extensions.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/coreos-layering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coreos-layering-configuring-on-extensions_{context}"]
+= Installing extensions into an on-cluster custom layered image
```

Comment:
Hey @mburke5678, we are changing how we handle extensions for the OCL case and will need to remove this section completely from the docs. Extensions will be handled the same way as we did without OCL, so no need to mention it here I believe.